controllers/comments-controller.js

```diff
@@ -0,0 +1,50 @@
+const Comment = require('../models/comment');
```

Comment:
I'm not sure if it's should call CommentModel to have the same name as we had in class. But I actually prefer your way